core/src/test/java/io/jenkins/pluginhealth/scoring/repository/ScoreRepositoryIT.java

```diff
@@ -0,0 +1,74 @@
+        final Plugin plugin1 = entityManager.persist(new Plugin("plugin-1", new VersionNumber("1.0"), "scm", ZonedDateTime.now()));
+        final Plugin plugin2 = entityManager.persist(new Plugin("plugin-2", new VersionNumber("1.0"), "scm", ZonedDateTime.now()));
+        final Plugin plugin3 = entityManager.persist(new Plugin("plugin-3", new VersionNumber("1.1"), "scm", ZonedDateTime.now()));
+
+        final Score oldScore1 = entityManager.persist(new Score(plugin1, ZonedDateTime.now().minusDays(1)));
+        final Score oldScore2 = entityManager.persist(new Score(plugin1, ZonedDateTime.now().minusMonths(1)));
+        final Score recentScore1 = entityManager.persist(new Score(plugin1, ZonedDateTime.now().minusHours(8)));
+        final Score recentScore2 = entityManager.persist(new Score(plugin1, ZonedDateTime.now()));
+        final Score recentScore3 = entityManager.persist(new Score(plugin1, ZonedDateTime.now()));
+        final Score recentScore4 = entityManager.persist(new Score(plugin1, ZonedDateTime.now()));
+        final Score recentScore5 = entityManager.persist(new Score(plugin1, ZonedDateTime.now()));
+
+        final Score oldScore21 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusMonths(1)));
+        final Score oldScore22 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusDays(5)));
+        final Score recentScore01 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusDays(3)));
+        final Score recentScore02 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusDays(2)));
+        final Score recentScore03 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusDays(1)));
+        final Score recentScore04 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusHours(15)));
+        final Score recentScore05 = entityManager.persist(new Score(plugin2, ZonedDateTime.now().minusDays(4)));
+
+        final Score oldScore31 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(7)));
+        final Score oldScore32 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(6)));
+        final Score recentScore001 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(5)));
+        final Score recentScore002 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(4)));
+        final Score recentScore003 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(3)));
+        final Score recentScore004 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(2)));
+        final Score recentScore005 = entityManager.persist(new Score(plugin3, ZonedDateTime.now().minusDays(1)));
+
+        long noOfRowsDeleted = repository.deleteOldScoreFromPlugin();
+        assertThat(noOfRowsDeleted).isEqualTo(6);
+
+        List<Score> remainingScores = entityManager.getEntityManager().createQuery("SELECT s FROM Score s", Score.class).getResultList();
```

Comment:
You could use the `findAll` method on the repository object.

Reply:
> You could use the `findAll` method on the repository object.

Done.